src/Core/MySQL/PacketsReplication.h

```diff
@@ -55,6 +55,25 @@ class BinlogDump : public IMySQLWritePacket
     BinlogDump(UInt32 binlog_pos_, String binlog_file_name_, UInt32 server_id_);
 };
 
+/// https://dev.mysql.com/doc/internals/en/com-binlog-dump-gtid.html
+class BinlogDumpGTID : public IMySQLWritePacket
+{
+public:
+    UInt64 binlog_pos;
```

Comment:
Seems like `binlog_pos` is always 4 and `binlog_file_name` is always an empty string. These fields are used only in `BinlogDumpGTID::writePayloadImpl`. Maybe remove these fields and write constants?

Reply:
removed

Reply:
binlog_file_name field removed